### PR TITLE
enable log rotation by default and expose backups parameter to user

### PR DESCRIPTION
**Pull request by sctb512**
## Relevant Issue (if applicable)
_If there are Issues related to this PullRequest, please list it._

## Details
_Please describe the details of PullRequest._

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: src/bin/nydusd/main.rs [hunk 1]
```diff
@@ -248,7 +248,15 @@ fn prepare_commandline_options() -> Command {
             Arg::new("log-rotation-size")
                 .long("log-rotation-size")
                 .help("Specify log rotation size(MB), 0 to disable")
-                .default_value("0")
+                .default_value("100")
```
Comment (changweige) [inline src/bin/nydusd/main.rs:251]: We should change the default value of nydusd which is somewhat of a UI. Changing the usual behavior is not good.

File: src/bin/nydusd/main.rs [hunk 2]
```diff
@@ -248,7 +248,15 @@ fn prepare_commandline_options() -> Command {
...
+                .required(false)
+                .global(true),
+        )
+        .arg(
+            Arg::new("log-rotation-backups")
```
Comment (changweige) [inline src/bin/nydusd/main.rs:256]: At this stage, giving the ability to configure `log-rotation-size` at nydus-snapshotter side is acceptable with the hardcoded 10 backups

File: src/bin/nydusd/main.rs (hunk 1 above)
Comment (sctb512) [reply src/bin/nydusd/main.rs:251]: Got it.

File: src/bin/nydusd/main.rs (hunk 2 above)
Comment (sctb512) [reply src/bin/nydusd/main.rs:256]: If we use the hardcoded 10 backups, there no need to expose `log-rotation-backups` to users. I will close this PR later. 😄

**Pull request closed without merging**